presto-main/src/main/java/com/facebook/presto/sql/gen/AndCodeGenerator.java

```diff
@@ -20,12 +20,14 @@
 import com.facebook.presto.bytecode.instruction.LabelNode;
 import com.facebook.presto.common.type.Type;
 import com.facebook.presto.spi.relation.RowExpression;
+import com.facebook.presto.spi.relation.SpecialFormExpression;
 import com.google.common.base.Preconditions;
 
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import java.util.Stack;
```

Comment:
Let's do not use `Stack` class. It is not recommended in Java docs. Use `ArrayDeque` for the impl.

Reply:
Done